src/views/landing/LandingPage.tsx

```diff
@@ -3,17 +3,24 @@ import { Box, Grid, Typography } from '@mui/material'
 import LandingPageAppWidget from './LandingPageAppWidget'
 import { useDispatch } from 'react-redux'
 import { useNavigate } from 'react-router'
+import { getActiveNetwork } from '../../redux/slices/network'
 import { changeActiveApp, getAllApps } from '../../redux/slices/app-config'
 import { useAppSelector } from '../../hooks/reduxHooks'
 
 export default function LandingPage() {
+    const activeNetwork = useAppSelector(getActiveNetwork)
     const dispatch = useDispatch()
     const navigate = useNavigate()
     const allApps = useAppSelector(getAllApps)
     const isAuth = useAppSelector(state => state.appConfig.isAuth)
     const handleWidgetClick = app => {
         dispatch(changeActiveApp(app?.name))
-        navigate(app?.url)
+
+        if (app.name === 'Explorer') {
```

Comment:
Same as above. You should use the network's id instead and not the name of the network